### PR TITLE
chore: migrate to pnpm

### PR DESCRIPTION
**Pull request by QuiiBz**
Migrate from `yarn` to `pnpm`

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Allow running `prepare` script from shire
```
Comment (chambo-e) [inline .npmrc:1]: sure ?

File: pnpm-workspace.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+packages:
```
Comment (chambo-e) [inline pnpm-workspace.yaml:1]: need to be removed from package.json ?

File: .npmrc (hunk 1 above)
Comment (QuiiBz) [reply .npmrc:1]: Option need but forgot to rename, here's without this `unsafe-perm`:

Screenshot 2022-03-08 at 11 54 38

File: pnpm-workspace.yaml (hunk 2 above)
Comment (QuiiBz) [reply pnpm-workspace.yaml:1]: Not working with lerna if we remove it

**Review by adriengibrat — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -98,75 +98,75 @@ scaleway-lib is a set of NPM packages used at Scaleway.
...
 
 ```bash
-$ yarn global add yalc # Make sure to have the yalc binary
+$ pnpm install -g yalc # Make sure to have the yalc binary
 ```
 
 ```bash
 $ cd scaleway-lib/packages/example_package
-$ yarn build && yalc publish
+$ pnpm build && yalc publish
```
Comment (chambo-e) [inline README.md:117]:
```suggestion
$ pnpm run build && yalc publish
```

**Review by chambo-e — CHANGES_REQUESTED** [review]

**Review by chambo-e — APPROVED** [review]

**Pull request merged**